include/zephyr/net/net_core.h

```diff
@@ -29,6 +29,8 @@ extern "C" {
 /**
  * @brief Networking
  * @defgroup networking Networking
+ * @since 1.0
+ * @version 1.0.0
```

Comment:
I don't think this should have any version information, as it is an umbrella entry without anything in the group. All the other network related groups are children of this "networking" group but it does not make them stable. If one looks at the generated documentation list, it gives an impression that all the network APIs are v1.0.0 which is not the case.